tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
-envlist = py{35,36,37,38}
-minversion = 3.2
+envlist = py{36,37,38}
+minversion = 3.6
```

Comment:
This refers to Tox version, not Python version.

Reply:
TIL